posthog/api/test/test_survey.py

```diff
@@ -832,6 +945,9 @@ def test_can_list_surveys(self):
         list = self.client.get(f"/api/projects/{self.team.id}/surveys/")
         response_data = list.json()
         assert list.status_code == status.HTTP_200_OK, response_data
+
+        # remove the automatically inserted targeting flag from test assertion.
```

Comment:
not sure what this means?

Reply:
I'll remove the comment, no longer necessary with the changes I made.